### ISSUE TITLE
False positive in colon rule with #imageLiteral

### ISSUE THREAD

**Issue by JoeFerrucci** [realm/SwiftLint#2740]
### New Issue Checklist

- [X] Updated SwiftLint to the latest version
- [X] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

In scenarios where there's a `colon` and an `#imageLiteral`, the linter warns.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
Colon Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)
```

### Environment

* SwiftLint version:`0.32.0`
* Installation method used: `CocoaPods`
* Configuration file:

```yml
file_length: 450

nesting:
  type_level: 3

swiftlint_version: 0.32.0

cyclomatic_complexity:
  ignores_case_statements: true
```

* Are you using nested configurations? No
* Xcode version:`Xcode 10.2 (10E125)`

```swift
// This triggers a violation:
toggleImageView.image = isExpanded ? #imageLiteral(resourceName:"grayArrowUp") : #imageLiteral(resourceName: "grayArrowDown")

// This also triggers a violation:
let imageView = UIImageView(image: #imageLiteral(resourceName:"grayArrowUp"))
```

**Comment by marcelofabri** [comment]
The warning is triggered inside `#imageLiteral`, where it's actually missing a space. Xcode 10.2 adds a space when inserting a literal, so I'm closing this bug.

Note that it might be the case that previous Xcode versions did not add that space.
